test.js

```diff
@@ -67,3 +67,9 @@ test('weird null issue', t => {
 test('support true color escape sequences', t => {
 	t.is(sliceAnsi('\u001B[[1m\u001B[[48;2;255;255;255m\u001B[[38;2;255;0;0municorn\u001B[[39m\u001B[[49m\u001B[[22m', 0, 3), '\u001B[1m\u001B[48;2;255;255;25m\u001B[38;2;255;0;0muni\u001B[39m');
 });
+
+test.failing('doesn\'t add extra escapes', t => {
+	const s = '\u001B[30m\u001B[43m RUNS \u001B[49m\u001B[39m  \u001B[32mtest\u001B[39m';
```

Comment:
Can you include the `chalk` call used to produce the string in a comment? And use a more descriptive variable name than `s`.